.gitignore

```diff
@@ -7,3 +7,5 @@ pdf/
 .idea/
 __pycache__/
 docs/
+Dockerfile
```

Comment:
Just double checking that this is needed in the .gitignore file.

Reply:
Yep. Mainly because it's on my personal machine. You probably would not have one in your local environment.